### PR TITLE
Fix for the pause menu "Save Game" button sound effect

### PR DESCRIPTION
**Pull request by TheMachineAmbassador**
"Save Game" sound effect changed with red square save points.

### CODE REVIEW

File: Common/Settings.h [hunk 1]
```diff
@@ -94,7 +94,8 @@
 	visit(WhiteShaderFix, true) \
 	visit(WidescreenFix, true) \
 	visit(WndModeBorder, false) \
-	visit(WoodsideRoom205Fix, true)
+	visit(WoodsideRoom205Fix, true) \
+	visit(SaveGameSoundFix, true)
```
Comment (Polymega) [inline Common/Settings.h:98]: This list should be kept in alphabetical order.

File: Common/Settings.ini [hunk 2]
```diff
@@ -280,6 +280,9 @@ WhiteShaderFix = 1
 ; Fixes spawning placements for objects in Woodside Apartments Room 205. (0|1)
 WoodsideRoom205Fix = 1
 
+; Fixes "Game Save" button audio. (0|1)
```
Comment (Polymega) [inline Common/Settings.ini:283]: Please place this fix below `RowboatAnimationFix` to keep in alphabetical order. Let's make its description a little more detailed:

```ini
; Restores the "game save" sound effect when selecting to save your game through the pause menu.
```

**Review by Polymega — COMMENTED** [review]
Great work again! I have two notes on the PR before Elisha reviews.

**Comment by elishacloud** [comment]
Nice work.  Thx!

**Pull request merged**